docs/contributing/database-migrations/edd.mdx

```diff
@@ -173,49 +222,82 @@ END
 </TabItem>
 </Tabs>
 
-## Workflow
+## Our EDD Process
+
+There are some unique constraints to Bitwarden that are not addressed directly in Martin Fowler's
+EDD article.
+
+- Our Production instances in the cloud are required to be on at all times
```

Comment:
⛏️ When possible I think it's a good idea to eliminate personal pronouns; there's a lot of "our" and "we" in this document.

Reply:
I removed all of the personal pronouns that I added. I did not update the documented process that already existed.